online-shop/src/Components/EditProductPageComponent/EditProductDumpView.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import '../../Styles/ComponentsStyles/ProductDetailsStyles/ProductDetails.scss';
+import { Link } from 'react-router-dom';
+import { EditProductPageProps } from './EditProductSmartView';
+import { ActionStatusReport } from '../../HelperComponents/ActionStatusReport';
+
+export const EditProductDumpView: React.FC<EditProductPageProps> = (props: EditProductPageProps) => {
+    if (!props.isLoading) {
+        if (props.submitChangesResponse === 0) {
```

Comment:
To decouple from the REST API, you could translate the status codes to an enum that you define in your UI, e.g.:

enum ResponseType {
  OK = 0,
  NOT_FOUND = 404,
  ...
}